src/components/scenes/PluginViewScene.js

```diff
@@ -176,11 +176,12 @@ class PluginView extends React.Component<Props> {
   }
 
   render () {
-    const { uri, originWhitelist = ['file://*', 'https://*', 'http://*', 'edge://*'] } = this.props.plugin
+    const { uri, addOnUrl, originWhitelist = ['file://*', 'https://*', 'http://*', 'edge://*'] } = this.props.plugin
```

Comment:
I would use a default, like `addOnUrl = ''`, rather than the `|| ''` later.

Reply:
Done.